src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -2240,7 +2240,7 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
         public override BoundNode VisitPropertyAccess(BoundPropertyAccess node)
         {
             var result = base.VisitPropertyAccess(node);
-            if (Binder.AccessingAutoPropertyFromConstructor(node, this.CurrentSymbol))
+            if (Binder.IsPropertyAssignableFromConstructor(node, this.CurrentSymbol)) // PROTOTYPE(semi-auto-props): Revise this method call is the behavior we want and add unit tests..
```

Comment:
>IsPropertyAssignableFromConstructor [](http://example.com/codeflow?start=23&length=35)

Similar comment about the name. However, here I think we are specifically dealing with read access.